spi/src/main/java/org/eclipse/microprofile/openapi/spi/OASFactoryResolver.java

```diff
@@ -55,7 +48,7 @@ public abstract class OASFactoryResolver {
      * @throws IllegalArgumentException
      *             if an instance could not be created, most likely, due to an illegal or inappropriate class
      */
-    public abstract <T extends Constructible> T createObject(Class<T> clazz);
+    public abstract <T> T createObject(Class<T> clazz);
```

Comment:
Dropped `extends Constructible` to avoid dependency cycle with `api` module.

Reply:
Implementations should already be checking that `T` is `Constructible` and the API used by applications in `OASFactory` retains the `Constructible` type on `createObject`.